master/getting-started/kubernetes/installation/app-layer-policy.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Installing Calico with Application Layer Policy
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/installation/app-layer-policy'
+---
+
+Application Layer Policy for {{site.prodname}} allows you to write policies that enforce against both network layer attributes, like IP addresses or ports and application layer attributes like HTTP Method or Path. It can also enforce policy based on cryptographically secure identities, allowing you to reduce trust in your network.  See [Application Layer Policy Introduction](link me) for more information.
```

Comment:
I think you are missing a comma between `ports` and `and`. That did make this sentence very difficult for me at first. 😄 Also, I think it would be better to lowercase "Method" and "Path." In addition, I think pluralizing "method" and "path" would be best.

Reply:
Second sentence: everything sounds great until I get to the last phrase. When I read "reduce trust in your network" it seems exactly backwards. I can appreciate that perhaps for _very_ sophisticated audiences, this might mean something good (?) but probably for the vast majority, not so much? Just my take on it! I am not quite sure what you're trying to say, exactly, or perhaps I could advise you on an alternative. In the meantime, I might just suggest dropping that phrase.